### PR TITLE
This is the work I've done on Nationalities

### PR DESCRIPTION
**Pull request by jemsyjose**

### CODE REVIEW

File: src/test/java/ca/qaguru/oranghrmbatch24/pages/NationalitiesPage.java [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+    private final String CancelDeleteNationButton="//*[@type='button'][@class='oxd-button oxd-button--medium oxd-button--text orangehrm-button-margin']";
+    private final String LogoutDropdownButton="//*[@class='oxd-userdropdown-name']";
+    private final String LogoutButton="(//*[@role='menuitem'][@class='oxd-userdropdown-link'])[4]";
+    private final String Banner="//*[@class='oxd-toast oxd-toast--success oxd-toast-container--toast']";
+    public NationalitiesPage(WebDriver driver)
+    {
+        super(driver);
+    }
+
+    public void AddButtonVisibleClickable()
```
Comment (RahulNair92) [inline src/test/java/ca/qaguru/oranghrmbatch24/pages/NationalitiesPage.java]: method name has to start with small letter

File: src/test/java/ca/qaguru/oranghrmbatch24/pages/NationalitiesPage.java [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+        driver.findElement(By.xpath(NationTextBoxField)).sendKeys(nationality);
+        new WebDriverWait(driver,Duration.ofSeconds(WAIT_TIME))
+                .until(ExpectedConditions.elementToBeClickable(By.xpath(SaveButton)));
+        click(By.xpath(SaveButton));
+        new WebDriverWait(driver,Duration.ofSeconds(wait_time))
+                .until(ExpectedConditions.visibilityOfElementLocated(By.xpath(Banner)));
+        boolean dis=driver.findElement(By.xpath(Banner)).isDisplayed();
+        System.out.println("green banner with SUCCESS message visible :"+dis);
+        return dis;
+       /*
```
Comment (RahulNair92) [inline src/test/java/ca/qaguru/oranghrmbatch24/pages/NationalitiesPage.java]: remove the comment

File: src/test/java/ca/qaguru/oranghrmbatch24/tests/NationalitiesTests.java [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+    @Test(priority = 7)
+    public void LogoutAdmin()
+    {
+        loginPage = new LoginPage(driver);
+        headerPage = new HeaderPage(driver);
+        nationalitypage = new NationalitiesPage(driver);
+        loginPage.login("Admin", "admin123", true, null);
+        nationalitypage.logoutDropdownMenuVisibleClickable();
+        nationalitypage.logout();
+        driver.close();
```
Comment (RahulNair92) [inline src/test/java/ca/qaguru/oranghrmbatch24/tests/NationalitiesTests.java]: remove driver.close

**Review by RahulNair92 — COMMENTED** [review]
please fix comments

File: src/test/java/ca/qaguru/oranghrmbatch24/pages/NationalitiesPage.java (hunk 1 above)
Comment (jemsyjose) [reply src/test/java/ca/qaguru/oranghrmbatch24/pages/NationalitiesPage.java]: Done

File: src/test/java/ca/qaguru/oranghrmbatch24/pages/NationalitiesPage.java (hunk 2 above)
Comment (jemsyjose) [reply src/test/java/ca/qaguru/oranghrmbatch24/pages/NationalitiesPage.java]: Done

File: src/test/java/ca/qaguru/oranghrmbatch24/tests/NationalitiesTests.java (hunk 3 above)
Comment (jemsyjose) [reply src/test/java/ca/qaguru/oranghrmbatch24/tests/NationalitiesTests.java]: Done

**Review by RahulNair92 — APPROVED** [review]
Approve my changes

**Pull request merged**